app/models/user.rb

```diff
@@ -110,6 +111,17 @@ def student?(course)
     course.users.role('student').include? self
   end
 
+  def campaign_organizer?
+    organized_campaigns.any?
+  end
+
+  def organized_campaigns
```

Comment:
So should this be moved to DashboardPresenter? There we want to show only campaigns the current user organizes. I just thought it was a nifty method that could exist on the model